event_sidecar/src/lib.rs

```diff
@@ -259,6 +259,7 @@ fn builder(
         ip_address: connection.ip_address,
         sse_port: connection.sse_port,
         rest_port: connection.rest_port,
+        network_name: connection.network_name.clone(),
```

Comment:
The idea is that the network name should be global, not connection-based. We want to prevent the user to accidentally connect to different networks.